src/wopiserver.py

```diff
@@ -311,6 +318,31 @@ def cboxOpen():
   return 'Client not authorized', http.client.UNAUTHORIZED
 
 
+@Wopi.app.route("/wopi/cbox/open", methods=['GET'])
+def cboxOpen():
+  '''CERNBox-specific endpoint for /open, provided for backwards compatibility'''
+  return iopOpen()
+
+
+@Wopi.app.route("/wopi/iop/open/list", methods=['GET'])
+def iopGetOpenFiles():
+  '''Returns a list of all currently opened files, for operations purposes only.
+  This call is protected by the same shared secret as the /wopi/iop/open call.'''
+  req = flask.request
+  # first check if the shared secret matches ours
+  if 'Authorization' not in req.headers or req.headers['Authorization'] != 'Bearer ' + Wopi.iopsecret:
+    Wopi.log.warning('msg="iopGetOpenFiles: unauthorized access attempt, missing authorization token" ' \
+                     'client="%s"' % req.remote_addr)
+    return 'Client not authorized', http.client.UNAUTHORIZED
+  # first convert the sets into lists, otherwise sets cannot be serialized in JSON format
+  jl = {}
```

Comment:
What does jl stand for?

Reply:
json-able list. OK, that's too short, I renamed it to `jlist`. Again it's almost a mute variable, generated there and sent over in the next line.